web/docs/operators/lookup.md

```diff
@@ -0,0 +1,83 @@
+# lookup
+
+Enriches a stream of events with a context.
+
+## Synopsis
+
+```
+lookup <context> [<options>] [--live] [--retro] [--snapshot]
+lookup <field=><context> [<options>] [--live] [--retro] [--snapshot]
+```
+
+## Description
+
+The `lookup` operator applies a context, extending input events with a new field
+defined by the context, then outputs only the events that contain this
+enrichment.
+
+### `<context>`
+
+The name of the context to lookup with.
+
+### `<field>`
+
+The name of the field in which to store the context's enrichment. Defaults to
+the name of the context.
+
+### `<options>`
+
+Optional, context-specific options. Refer to the [`enrich` operator
+documentation](../transformations/enrich.md) for more details about these
```

Comment:
This link has moved. Same for the others below.

Reply:
Fixed in https://github.com/tenzir/tenzir/pull/3721/commits/9d6e1412952dfe0ac4724fb7199cf4096375179a